lib/twine/placeholders.rb

```diff
@@ -24,7 +24,7 @@ def convert_placeholders_from_twine_to_android(input)
       # %@ -> %s
       value = convert_twine_string_placeholder(input)
 
-      number_of_placeholders = number_of_twine_placeholders(input)
+      number_of_placeholders = number_of_twine_placeholders(value)
```

Comment:
oh damn, that looks like a pretty stupid bug 🤦‍♂️ Thanks for the quick response and fixing it!